src/components/DaySlot.jsx

```diff
@@ -0,0 +1,225 @@
+import React, { PropTypes, Component } from 'react'
+import { findDOMNode } from 'react-dom'
+import cn from 'classnames';
+
+import SelectableTimeGutter from '../containers/SelectableTimeGutter.jsx'
+import EventRow from '../EventRow.jsx'
+
+import dates from '../utils/dates';
+import { isSelected } from '../utils/selection';
+import { notify } from '../utils/helpers';
+import { accessor, elementType } from '../utils/propTypes';
+import { accessor as get } from '../utils/accessors';
+import localizer from '../localizer'
+
+export default class DaySlot extends Component {
+  static propTypes = {
+    formatter: PropTypes.func,
```

Comment:
what is it formatting? if its date related then it more correctly should be a date format prop, which has a hook into the localizer. Also in case you missed you it a user can customize formatting already, as formats can also be functions.